lib/Compiler.js

```diff
@@ -56,6 +56,8 @@ class Compiler extends Tapable {
 			emit: new AsyncSeriesHook(["compilation"]),
 			/** @type {AsyncSeriesHook<Compilation>} */
 			afterEmit: new AsyncSeriesHook(["compilation"]),
+			/** @type {AsyncSeriesHook<Compilation>} */
+			assetEmitted: new AsyncSeriesHook(["compilation"]),
```

Comment:
The `hooks` object was lacking `assetEmitted` key. How should it be refactored :thinking: